<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphs using Qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 600 10pt &quot;Segoe UI Semibold&quot;;</string>
        </property>
        <property name="text">
         <string>        X AXIS :-    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxXAxis">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">
font: 600 10pt &quot;Segoe UI Semibold&quot;;</string>
        </property>
        <property name="text">
         <string>        Y AXIS :-   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxYAxis">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);
</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="GreenIndicator">
        <property name="toolTip">
         <string>start indicator</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RedIndicator">
        <property name="toolTip">
         <string> stop indicator</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="LoadfileButton">
        <property name="toolTip">
         <string>Select file to plot</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);</string>
        </property>
        <property name="text">
         <string>    Select Files    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 600 10pt &quot;Segoe UI Semibold&quot;;</string>
        </property>
        <property name="text">
         <string>       File Path   :-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_FileName">
        <property name="toolTip">
         <string>File path</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color:rgb(0, 0, 127);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CaptureData">
        <property name="toolTip">
         <string>Click to start capturing data</string>
        </property>
        <property name="toolTipDuration">
         <number>-4</number>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);</string>
        </property>
        <property name="text">
         <string>  Save  Data </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCustomPlot" name="customplot1" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="customplot2" native="true"/>
      </item>
      <item>
       <widget class="QCustomPlot" name="customplot3" native="true"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxGraph1">
        <property name="toolTip">
         <string>view sensor 1 data</string>
        </property>
        <property name="toolTipDuration">
         <number>-5</number>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:rgb(0, 0, 112);
font: 600 12pt &quot;Segoe UI Semibold&quot;;
</string>
        </property>
        <property name="text">
         <string>Sensor 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGraph2">
        <property name="toolTip">
         <string>view sensor 2 data</string>
        </property>
        <property name="toolTipDuration">
         <number>-3</number>
        </property>
        <property name="styleSheet">
         <string notr="true">
color:rgb(218, 0, 0);

font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
        </property>
        <property name="text">
         <string>Sensor 2</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxGraph3">
        <property name="toolTip">
         <string>view sensor 3 data</string>
        </property>
        <property name="toolTipDuration">
         <number>-3</number>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(73, 145, 0);

font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
        </property>
        <property name="text">
         <string>Sensor 3</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCenter1">
        <property name="toolTip">
         <string>Click to set default scale for graph1</string>
        </property>
        <property name="toolTipDuration">
         <number>-4</number>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);</string>
        </property>
        <property name="text">
         <string>Re-Center (S1)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCenter2">
        <property name="toolTip">
         <string>Click to set default scale for graph1</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);
</string>
        </property>
        <property name="text">
         <string>Re-Center (S2)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCenter3">
        <property name="toolTip">
         <string>Click to set default scale for graph1</string>
        </property>
        <property name="toolTipDuration">
         <number>-4</number>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(214, 214, 214);
font: 12pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Re-Center (S3)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ClearGraph">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background:rgb(214, 214, 214);</string>
        </property>
        <property name="text">
         <string>  Clear Graph  </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionPort_Settings"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionPort_Settings"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionPort_Settings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/settings-icon.png</normaloff>:/images/settings-icon.png</iconset>
   </property>
   <property name="text">
    <string>Port Settings</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
